src/QsCompiler/Core/SymbolTable.fs

```diff
@@ -1342,16 +1427,16 @@ and NamespaceManager
     member this.NamespacesContainingCallable cName = 
         // FIXME: we need to handle the case where a callable/type with the same qualified name is declared in several references!
         syncRoot.EnterReadLock()
-        try let containsCallable (ns : Namespace) = ns.ContainsCallable cName |> QsNullable<_>.Map (fun _ -> ns.Name)
-            (Namespaces.Values |> QsNullable<_>.Choose containsCallable).ToImmutableArray()
+        try let tryFindCallable (ns : Namespace) = ns.TryFindCallable cName |> QsNullable<_>.Map (fun _ -> ns.Name)
+            (Namespaces.Values |> QsNullable<_>.Choose tryFindCallable).ToImmutableArray()
         finally syncRoot.ExitReadLock()
 
     /// Returns the names of all namespaces in which a type with the given name is declared. 
```

Comment:
It would be good to mention whether or not private/internal things are included.
Edit: As discussed, the behavior of these should probably be changed (I believe they are only used for a code action, but you need to double check that). If you want to do that in a separate PR, then please add a comment on NamespacesContainingType and NamespacesContainingCallable.